packages/test/src/test-dependencies.ts

```diff
@@ -218,35 +213,15 @@ if (RUN_INTEGRATION_TESTS) {
     const p = worker.run();
     const conn = new WorkflowClient();
     const wf = conn.stub<{ main(): number }>('ignored-dependencies', { taskQueue });
-    const runId = await wf.start();
+    await wf.start();
     const result = await wf.result();
     worker.shutdown();
     await p;
 
-    const info: WorkflowInfo = {
-      namespace: 'default',
-      taskQueue,
-      workflowId: wf.workflowId,
-      runId,
-      filename: 'ignored-dependencies',
-      isReplaying: false,
-    };
-
     t.deepEqual(
       thrownErrors.map((err) => err.message),
       ['syncIgnored.syncImpl', 'syncIgnored.asyncImpl', 'asyncIgnored.syncImpl', 'asyncIgnored.asyncImpl']
     );
-    t.deepEqual(
-      recordedCalls,
-      thrownErrors.map((error) => ({
-        level: 'ERROR',
-        message: 'External dependency function threw an error',
-        meta: {
-          error,
-          workflowInfo: info,
-        },
-      }))
-    );
```

Comment:
I figured it was fine to remove this since we're already checking the exceptions get thrown. This is the only test that wants to intercept the logger this way so it felt meh to have to keep that around somehow just for it.

Reply:
If only I'd documented why this is tested...

Let's remove this for now, we can add it back later.

Reply:
If you don't mind, please add it back